src/components/todos/TodoItem.js

```diff
@@ -21,20 +56,22 @@ class TodoItem extends Component {
       });
     }
 
+    console.log(todo);
+    
+
     return (
```

Comment:
I would structure the html a little differently. Bootstrap columns would be great to utilize in this instance, maybe something like:

```
<div className="row">            
    <div className="col-11">
         input    
     </div>
     <div className="col-1">
         icon
    </div>
</div>
```